AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -8,6 +8,9 @@
 ! https://github.com/AdguardTeam/AdguardFilters/issues/76987
 youtube.com#%#AG_onLoad(function() {var a = new MutationObserver(function() {if (document.querySelector('yt-upsell-dialog-renderer')) {var b = document.querySelector('.ytd-popup-container[popup-size="UPSELL_DIALOG_RENDERER_POPUP_SIZE_LARGE"] #dismiss-button:not(.dismissed), .upsell-dialog-lightbox .upsell-dialog-dismiss-button button:not(.dismissed)');setTimeout(function() {b&&b.click()}, 3E2)}});a.observe(document, {childList: !0,subtree: !0});setTimeout(function() {a.disconnect()}, 3E3)});
 !
+! https://github.com/AdguardTeam/AdguardFilters/pull/83911
```

Comment:
Dangerous. These scripts are not used to block some events only.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/83911#discussion_r639919898
I know. I have tried to block/use JavaScript method to prevent that.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/83911/commits/df5cf57a0af7d9b0e3df34c2e838a36c7733a015